models/mock_model.py

```diff
@@ -87,16 +88,16 @@ def on_press(self, key):
                 self.enter = 1
             case Key.right:
                 self.forward = 1
-                self.mode_index = (self.mode_index + 1) if self.mode_index < 6 else 0
+                self.mode_index = (self.mode_index + 1) if self.mode_index < 5 else 0
             case Key.left:
                 self.backward = 1
-                self.mode_index = (self.mode_index - 1) if self.mode_index > 0 else 6
+                self.mode_index = (self.mode_index - 1) if self.mode_index > 0 else 5
```

Comment:
this code is bad, just do the length of the modes or something. this is not good to change every time we add or remove a mode (and pls fix everywhere)